style_rwd.css

```diff
@@ -1,22 +1,67 @@
 @media (max-width: 767px){
     img.main_photo {
         display: none;
+        width:auto;
     }
-    .catch_phrase{
-        max-width: 767px;
+    .section_hero {
+        padding-bottom: 42px;
+    }
+    body{
+        width: auto;
     }
     .column{
+        width: auto;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
+        font-size: 40px;
         font-weight: 30;
-        max-width: 767px;
+        margin-bottom: 0;
+        width: auto;
+    }
+    .button_container {
+        justify-content: center;
+        width: auto;
+        margin: 0;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
+        padding: 5px 16px;
         font-size: 12px;
     }
-}
+    .section_container {
+        min-height: 400px;
+        justify-content: center;
+    }
+    .catch_phrase {
+        flex-direction: column;
+        width: auto;
+        line-height: 95px;
+    }
+
+    .flex_column {
+        flex-direction: column;
+    }
+
+    .section_heading {
+        margin: auto;
+        font-size: 18px;
+        font-weight: 200;
+        width: auto;
+        padding-top: 17px;
+    }
+    .pros_container {
+        width: auto;
+        margin: auto;
```

Comment:
you don’t want a margin auto here.

Reply:
is it better now? Why margin auto is not correct in here?

Reply:
best practice: `margin: 0 auto;`

Reason:


`The body's height (which is typically auto, not 100%) has no effect on how auto vertical margins are calculated. The spec simply defines auto vertical margins to always be zero on in-flow block boxes. So for in-flow block boxes, the answer to the question is not yes, but no - there is no difference between margin: 0 auto and margin: auto. The answer is only yes in one other specific case: absolutely positioned elements. –`